modules/dr-restoring-cluster-state.adoc

```diff
@@ -90,7 +90,37 @@ $ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard"
 +
 The output of this command should be empty. If it is not empty, wait a few minutes and check again.
 
-.. Move the etcd data directory to a different location:
+.. Move the existing Kubernetes Controller Manager pod file out of the kubelet manifest directory by using the following command:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/kube-controller-manager-pod.yaml /tmp
+----
+
+.. Verify that the Kubernetes Controller Manager containers are stopped by running the following command:
++
+[source,terminal]
+----
+$ sudo crictl ps | grep kube-controller-manager | egrep -v "operator|guard"
+----
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
```

Comment:
The IBM SG states:

![Screenshot from 2024-01-29 12-23-34](https://github.com/openshift/openshift-docs/assets/57954076/88342245-a9b8-48c1-b429-f285a203fae4)


How about: 

```suggestion
Empty output is the desired result from running the command. If the output is not empty, wait a few minutes and check again.
```
WDYT?